.github/workflows/spec.yml

```diff
@@ -0,0 +1,38 @@
+name: "Static & Spec Tests"
+
+on: [pull_request, workflow_dispatch]
+
+jobs:
+  StaticAndSpecTests:
+    name: "Static & Spec Tests (Ruby ${{ matrix.ruby_version }})"
+    strategy:
+      fail-fast: false
+      matrix:
+        ruby_version: ['2.5', '2.7']
+    runs-on: ubuntu-latest
+    
+    steps:
+    - name: Checkout Source
+      uses: actions/checkout@v2
+
+    - name: "Activate Ruby ${{ matrix.ruby_version }}"
+      uses: ruby/setup-ruby@v1
+      with:
+        ruby-version: ${{ matrix.ruby_version }}
+        bundler-cache: true
+        without: "development"
+
+    - name: Rubocop
+      run: bundle exec rake rubocop
+
+    - name: 'Spec:Coverage'
+      run: bundle exec rake spec:coverage
```

Comment:
There's a silent failure in this step for connecting to coveralls. With us still being connected with travis, I'm not sure if this is something that can be changed?

Reply:
Good point, I'll maybe look in to the Coveralls Github Action: https://github.com/marketplace/actions/coveralls-github-action

Reply:
...but yeah, it may not like this being reported in from two sources simultaneously

Reply:
Given we decided to go for the [codecov Github Action](https://github.com/puppetlabs/pdk/pull/1030#issuecomment-807061849), we can safely have coveralls deprecated the same time we bin `.travis.yml`